jakarta.jdt/org.eclipse.lsp4jakarta.jdt.test/projects/jakarta-sample/src/main/java/io/openliberty/sample/jakarta/di/MultipleConstructorWithInject.java

```diff
@@ -0,0 +1,38 @@
+/*******************************************************************************
+* Copyright (c) 2021 IBM Corporation.
+*
+* This program and the accompanying materials are made available under the
+* terms of the Eclipse Public License v. 2.0 which is available at
+* http://www.eclipse.org/legal/epl-2.0.
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Contributors:
+*     Ananya Rao
+*******************************************************************************/
+
+package io.openliberty.sample.jakarta.di;
+
+import jakarta.inject.Inject;
+
+public class productIdentity{
+	private int productNum;
+	private String productDesc;
+	
+	@Inject
+    public productIdentity(int productNum) {
+		this.productNum = productNum;
+	}
+
+	@Inject
```

Comment:
Formatting looks slightly off in this file
Configure your Eclipse to use tabs instead of spaces:
https://newbedev.com/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs

Then run the formatter against this class:
https://www.eclipse.org/pdt/help/html/formatting_code.htm#:~:text=Open%20the%20required%20file.,press%20Ctrl%2BShift%2BF